app/views/backend/fixed_assets/index.html.haml

```diff
@@ -20,5 +20,9 @@
     = link_to :account_for.tl, { action: :depreciate_all, until: depreciate_until.iso8601, redirect: request.fullpath }, { class: 'btn btn-default', data: { confirm: :are_you_sure.tl, update_link_with: '#depreciate-fixed-assets-until' }, method: :post }
     %span.add-on= :up_to.tl.downcase
     = date_field_tag(:until, depreciate_until, id: 'depreciate-fixed-assets-until', data: { max_date: Time.zone.today.iso8601 })
+  = dropdown_menu_button(:export.ta, icon: :download) do |d|
+    -# %i[general by_account gain_and_loss].each do |state|
+    - %i[general by_account].each do |state|
```

Comment:
Can we use like.. `nature` instead of `state`? or `kind`? The keyword `state` implies it's a particular.. well state, of the financial years involved.